changelog/gc_registry.dd

```diff
@@ -0,0 +1,32 @@
+User supplied garbage collectors can now be linked with the runtime
+
+A GC registry has been implemented that allows to add
+garbage collector implementations by just linking them into
+the binary. To do so add a function that is executed before the
+D runtime initialization using `pragma(crt_constructor)`:
+
+---
+extern (C) pragma(crt_constructor) void registerMyGC()
+{
+    registerGCFactory("mygc", &createMyGC);
```

Comment:
Doesn't this need an import?

Reply:
It does (`import gc.registry, gc.gcinterface;`), but they are not easily accessible right now